plugins/modules/ipaconfig.py

```diff
@@ -418,12 +418,12 @@ def main():
                 if ansible_module.argument_spec.get(k):
                     arg_type = ansible_module.argument_spec[k]['type']
                     if k in (
-                           'ipaselinuxusermaporder', 'domain_resolution_order'
-                       ):
+                        'ipaselinuxusermaporder', 'domain_resolution_order'
+                        ):
```

Comment:
This format is both weird, and makes flake8 complains about the line below the test condition starting at the same indentation (also, it makes code harder to read, in my opinion).
What do you think about something like:
```python
if (
    k in ('some', 'options')
):
    # here goes the code block.
```
or if 'k in ()' being too long:
```python
if (
    k in (
        "some",
        "really long",
        "options",
    )
):
```

Reply:
Yes, this is somewhat "black-like" formatting, but this is a hard one. ;-)